styles/widgets/common/popup.less

```diff
@@ -24,12 +24,26 @@
 }
 
 .dx-popup-content {
+    box-sizing: content-box;
```

Comment:
Why???

Reply:
You broke User layouts

Reply:
As i understand, setContentHeight must compensate differences

Reply:
You'll check it?

Reply:
You'll check it?